### ISSUE TITLE
idea: "information for families" letter

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1367]
Here's an example of how IM does this:
https://curriculum.illustrativemathematics.org/MS/teachers/family_info.html

I think this is a pretty standard thing offered, which might be especially helpful for our curricula because we can't assume that families know about or understand what we are offering and how it is unique.

**Comment by flannery-denny** [comment]
@schanzer Thoughts on where this would live in the repository?

**Comment by schanzer** [comment]
@flannery-denny 

`shared/langs/en-us/docroot/`

**Comment by flannery-denny** [comment]
@retabak @schanzer Having read the IM one pager, I am reminded of how different our materials are from other materials out there... Their letter looks like the kind of letter I used to write to parents about _my classroom_.  And an IM teacher could print this letter in lieu of writing a letter about their classroom, because they'd bought into IM for the year. Bootstrap didn't follow me through the entire school year, so I wouldn't send out a comparable letter from Bootstrap at the beginning of the year.  More likely I'd email a link to it to parents who asked, or perhaps I'd email it out at a certain time of year... or maybe I wouldn't email it out at all... and some parents might go looking for it.

The phrases I like most from the IM letter are:
- carefully crafted and sequenced problems. 
- They gain experience communicating their ideas both verbally and in writing, developing skills that will serve them well throughout their lives. _This feels hyperbolic, but perhaps there's something to it?_
- This kind of instruction may look different from what you experienced in your own math education. 
- Current research says that... 

**Question: Is there any particular research we want to highlight in this letter?**

I suggest we start by getting clear about what we actually want to communicate for Bootstrap:Algebra and then branch out from there.

If I hadn't read the IM letter first, I think the goal of the letter would have been to respond to

**My child is learning computer science in math class?**

1. an opportunity to answer when am I ever going to use this?!
2. what is integrated cs
3. some data about the doors cs skills open?
4. some data about math transfer

**What else would you want to address in this letter?**

IM also asks the following 2 questions... do we want to follow that format? Or is some of this info more important to communicate than others and should we get at it with a different structure?

**What supports are in the materials to help my student succeed?**

1. Circles of Evaluation
2. Design Recipe
3. Error Messages

**What can my student do to be successful in this course?**

1. Keep track of contracts
2. Leave themselves comments in their game file
3. Make sure to write multiple examples for each function they're defining and follow the steps of the design recipe.

_FD - This last list isn't really doing it for me. And the broader kinds of answers IM gives aren't really relevant to us._

**Comment by retabak** [comment]
I'm excited you're writing and thinking deeply about this! FWIW, I would probably send it out to parents on the first day that we started coding. (I didn't do anything like this when I was in the classroom, but I wish that I did!)

I totally agree with the 4 items under "my child is learning computer science in math class?"

Regarding the supports - I like including these, but I think we'll need to be careful not to get too in the weeds. I wonder if we can concisely and meaningfully communicate how Circles of Evaluation, the design recipe, and error messages are special?

The items under "what can my student do to be successful?" feel even more specific than the supports, and I'm not sure if they have a place in a parent letter. Like, can we get a parent to understand these without writing an essay?

I wonder if the advice / supports could be presented in broader terms. Like, instead of saying, "Keep track of contracts," we could explain that, "although this is a computer science course, we do lots of work with paper and pencil! Your child will benefit from taking thorough notes about the computer science concepts they study." Or, instead of leave comments/ write examples, we could say: "Sometimes students get so excited about creating a video game, that they focus exclusively on making the game work. Instead, we want students to learn how to write code can be understood by another programmer."

**Comment by schanzer** [comment]
@flannery-denny thanks for the reminder! I say move forward with Rachel's feedback - I totally agree with her feedback.

**Comment by flannery-denny** [comment]
@schanzer @retabak I've pushed a draft for Algebra. Feedback or edits are welcome. Once we land on something we're satisfied with I can turn my attention to a Data Science letter.

**Comment by schanzer** [comment]
@flannery-denny made some edits and added some links (we should be touting our research as much as possible!).

I must admit I was surprised that you didn't mention our math learning outcomes. I added some prose to that effect, but if that was an intentional choice let's talk about it.

I also noticed that you put a ton of emphasis on the video game, to the exclusion of all other projects (flags, sam, the rocket, etc). Many teachers leave the video game out, so it feels strange to focus so heavily on just that one thing.

**Comment by flannery-denny** [comment]
@schanzer 

- I hadn't added links because I thought the document should stand on its own as some teachers are likely to print it. 
- I included the math learning outcomes in different locations than you did because I was trying to encourage parents to keep reading by writing as accessibly as possible
- Looking at the version you pushed, I decided to move the research paragraph below the bulleted list.
- I think links to Bootstrap:Algebra and Data Science looked fine, but the paragraph on research was quite visually cluttered so I shifted it to just a single link to our research page.
- as for mention of video games, I was lumping Sam and Rocket-height in with that. Perhaps you have a better overarching term? 
- I think that teachers who have their students build video games are much more likely to give out this letter than teachers who don't, in that they are devoting more time to it. 

Finally, noting that the way that @ds26gte added `docroot` `.adoc` files being converted to `.shtml` files during make requires `force-make`. Not sure whether or not that's how you think it should work, but it surprised me.

**Comment by schanzer** [comment]
@flannery-denny these changes all look fine to me. As for the links: just because _most_ people will print the document doesn't mean it's not worth making it maximally-useful for those who see it online. And printed links are the modern-day citations: even if people never bother to look them up, they communicate a sense of depth that I think is worthwhile.

If this is going to be so deeply linked to Algebra, I'm wondering if it makes sense just to move it into the algebra teacher resources page. Surely Data Science will have the same narrow audience, no?

**Comment by flannery-denny** [comment]
I’d guess that we want this page to be discoverable by parents so it should live on the teacher resources page AND somewhere else. What do you think @retabak?

**Comment by schanzer** [comment]
The TR pages folder is mostly about where things are stored in the repo. We can link to it from anywhere (including the) landing pages and/or the pathway narrative pages.

**Comment by retabak** [comment]
Hi! Just taking a look at this. I made some very minor changes to the file (check them out, let me know if you have questions / concerns). Here are someother thoughts:

- It's distracting to me that we have quotations in the last paragraph of the first section, but no real citations. (One can assume these quotes come from one of the articles that we've hyperlinked to, but obviously, that's not adequate.) I don't think there's any benefit to including quotations here. My instinct is to remove quotes around both phrases. (I'm assuming we're quoting Shriram / Kathi / Emmanuel and that this is okay???) 

- In the second section, we say we've designed the learning environment to do 4 things, but only 2 of the things (the first 2) relate to the programming environment. Can we include a different sentence preceding the last two bullets? Something like, "Our student workbooks also support the learning. We've developed workbook pages to: "

- In the last section: perhaps it's just personal thing, but I take issue with single-item lists. (They're not lists, so no bullets needed!) I've revised accordingly, but if you feel strongly about bullets... feel free to change.

**Comment by flannery-denny** [comment]
@retabak adding you to this because I'm pretty sure you took notes on this letter during our curriculum meetings and were going to finalize it?  I'm going to start tackling a DS letter.

**Comment by retabak** [comment]
Hey! I do have notes. Do you mind if I tackle this after PD in Tulsa (which
I'm nervous about / preparing for)?

If not, here are my notes
<https://docs.google.com/document/d/18azqRR1_y-hATRY91suZeEFMpdxoVNRqmRLBt0DB-CM/edit?usp=sharing>
!

**Comment by flannery-denny** [comment]
No rush! Just confirming that I remembered correctly that it was in your wheelhouse since we forgot to "assign" it to you at the meeting.

**Comment by retabak** [comment]
Hey, @flannery-denny - I updated this with the changes we discussed at our last curriculum meeting. Forgot to tag the commit to this issue (sorry).

**Comment by flannery-denny** [comment]
hey @retabak, I saw your commit last week and pushed a response. Please build the letter and confirm that it works for you.

**Comment by flannery-denny** [comment]
@schanzer @retabak

-  I am closing this issue as the algebra letter is completed and neither Rachel nor I are clear on what the big unifying question that a DS letter would be answering is. 
- I am moving the letter from docroot to algebra resources.
- We can make a new issue for a DS letter if someone gets clarity around what the focus question would be.

**Comment by schanzer** [comment]
Sounds good to me - can you make sure the letter is linked from somewhere sensible?

**Comment by flannery-denny** [comment]
@ds26gte I'm not able to figure out how to get this page moved and building properly. Please help!

The link needs to appear on `Pathways/Algebra-Pyret/../resources/index.adoc` and `Pathways/Algebra-WeScheme/../resources/index.adoc`

On both pages you will find

> == Printables for Classroom Walls
> 
> * @link{../../../lessons/computing-needs-all-voices/pages/computing-pioneers.html, Pioneers in Computing and Mathematics Posters}

Directly beneath that, we would like to add the following:

> == Information for Families
> 
> * @link{../../../info-for-families-alg.adoc, A letter you can share with families about Bootstrap:Algebra}
> 

The file is currently in `shared/docroot/info-for-families-alg.adoc`

We agreed that it should live in the algebra pathway. 

If possible, it would only live in one of the algebra pathways and be linked to from both.

For my first attempt I moved it to algebra-pyret and tried linking with `@link`
For my second attempt I tried linking with `@printable-exercise`
For my third attempt I moved it back to docroot and tried linking with `@link`
For my fourth attempt I tried linking with `@printable-exercise` to the docroot location

**The problem:** 

1. For all 4 attempts, the link did not render beautifully on the resource page:
<img width="299" alt="Screen Shot 2023-06-21 at 7 56 28 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/d13a1fd6-e686-4ed5-8628-13aedfef81be">

2. For all 4 attempts the link linked to adoc rather than the html. 
<img width="816" alt="Screen Shot 2023-06-21 at 7 56 38 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/c4855141-62db-40f8-887d-728ccb27f35f">

3. Since you're going to work on it anyway, we might as well optimize and provide a link to html as well as a link to pdf.

Thanks in advance!

**Comment by flannery-denny** [comment]
@ds26gte  Please have a look at the resources file in algebra-wescheme and see if you can get the link to information for families working. I have tried everything I can think of with ../../ etc. Thanks!
